app/src/main/java/com/framgia/soundclound/util/designhelper/GridSpacingItem.java

```diff
@@ -0,0 +1,57 @@
+package com.framgia.soundclound.util.designhelper;
+
+
+import android.content.Context;
+import android.graphics.Rect;
+import android.support.v7.widget.RecyclerView;
+import android.util.DisplayMetrics;
```

Comment:
ERROR: Unused import: android.util.DisplayMetrics. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports